### ISSUE TITLE
Replace people instead of clicking them out of the event

### ISSUE THREAD

**Issue by EllisDijkstra11** [saproto/saproto#1619]
P: Whenever someone has arranged to take over a spot specifically with someone else, the new participant is often low on the back-up list. If I add them to the event, they will not be on the back-up list, but the event will have another participant (the participant count increases by one). Deleting the original participant, however, then allows someone else to join the event. People on the back-up list do not receive the event emails

S: When someone is clicked out of the event, add a third option: confirm, cancel, and replace. If replace is chosen, a new name can be entered, who will take the previous place in the event (and if necessary, is removed from the back-up list).

Love you either way

**Comment by ysbrandB** [comment]
I agree that this would be a nice feature but in the meanwhile recently I fixed that if you click people out of an event which is too full nobody of the backuplist is put into the event. This means you can achieve the same thing as of right now by putting someone into a full event so that the participants becomes one too many for the participant limit and then clicking the person you want to signout out out of the event.
